terraform-infra-approvals/ccd-shared-infrastructure.json

```diff
@@ -3,6 +3,7 @@
   "module_calls": [
     {"source":  "git@github.com:hmcts/cnp-module-adf?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-waf?ref=ccd/CHG0033576"},
```

Comment:
is this still needed?

Reply:
will remove once this PR has merged https://github.com/hmcts/ccd-shared-infrastructure/pull/41